pylint_nautobot/status_field_instead_of_status_model.py

```diff
@@ -0,0 +1,26 @@
+"""Check for deprecated StatusModel usage and encourage the usage of StatusField instead."""
+from pylint.checkers import BaseChecker
+from pylint.interfaces import IAstroidChecker
+
+
+class NautobotDeprecatedStatusModelChecker(BaseChecker):
+    """Discourage the usage of deprecated StatusModel and encourage the usage of StatusField."""
+
+    __implements__ = IAstroidChecker
+
+    version_specifier = ">=2,<3"
+
+    name = "nautobot-status-field-instead-of-status-model"
+    msgs = {
+        "E4292": (
+            "Declare status on the model explicitly with a `StatusField`, as `StatusModel` is deprecated",
```

Comment:
Can you revert this to what it was before? The first string always describes the error, not what you should do instead.

```suggestion
            "Inherits from the deprecated StatusModel instead of declaring status on the model explicitly with StatusField",
```